src/Forms/SudoModePasswordField.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace SilverStripe\Forms;
+
+use SilverStripe\Forms\FormField;
+
+class SudoModePasswordField extends FormField
```

Comment:
Why not extend `PasswordField`?

Reply:
I cannot remember the exact reason. It did extended PasswordField at first, though something about that wasn't great so I changed it to FormField.

Reply:
Can you please try it again and take a note of what goes wrong? "Something about that wasn't great" doesn't help me review this - I need to know why it doesn't work so I can determine if there's another way around the problem you were facing, and if this is the correct approach or not.

Reply:
Seems fine, have updated to extended PasswordField.  Have added support for passing through the 'autocomplete' config on PasswordField to the SudoPasswordField react component